bats/sql-shell.bats

```diff
@@ -27,6 +27,19 @@ teardown() {
     [[ "$output" =~ "pk" ]] || false
 }
 
+@test "sql shell writes to disk after every iteration (autocommit)" {
```

Comment:
This isn't testing what you think it is. The expect script ends, which means the shell exits. You need to spawn a new shell while the old one is still running (or otherwise issue SQL queries).